tests/conformance/common.go

```diff
@@ -167,6 +167,10 @@ func ParseConfigurationMap(t *testing.T, configMap map[string]interface{}) {
 				val = uuid.New().String()
 				t.Logf("Generated UUID %s", val)
 				configMap[k] = val
+			} else if strings.Contains(val, "${{") {
+				s := strings.TrimSpace(strings.TrimSuffix(strings.TrimPrefix(val, "${{"), "}}"))
```

Comment:
What if you have `${{bla}} ${stuff}}`? This would fail then.

Reply:
I will accept this for now -- but I will break the moment someone tries to use two variables. So I'm with @ItalyPaleAle that we should use Go templates instead.